sdk/cosmosdb/cosmos/review/cosmos.api.md

```diff
@@ -301,6 +309,49 @@ export class ClientContext {
     }): Promise<Response_2<T & U & Resource>>;
 }
 
+// @public
+export class ClientEncryptionIncludedPath {
+    constructor(path: string, clientEncryptionKeyId: string, encryptionType: EncryptionType, encryptionAlgorithm: EncryptionAlgorithm);
+    clientEncryptionKeyId: string;
```

Comment:
can it be just id, algorithm, type ?

Reply:
Other SDKs have this naming convention. Let's keep consistency across SDKs.